perf_tests/cp_perf_test.py

```diff
@@ -6,7 +6,7 @@
     import pandas as pd
     import time
 
-    from lifelines.estimation import CoxPHFitter
+    from lifelines.fitters.coxph_fitter import CoxPHFitter
```

Comment:
hm, does `from lifelines import CoxPHFitter` still work?

Reply:
that's the "new" api I want to standardize, and get rid of the "old" `.estimation` syntax

Reply:
Oooh didn't try that.  The stuff in __init__ implies it should.  I'll have a go and update if so